opencti-platform/opencti-front/src/private/components/data/Root.jsx

```diff
@@ -109,8 +104,8 @@ const Root = () => {
           path="/dashboard/data/processing"
           render={() => (
             <Security
-              needs={[SETTINGS_SETACCESSES]}
-              placeholder={<Redirect to="/dashboard/data/processing/tasks" />}
+              needs={[KNOWLEDGE_KNUPDATE, SETTINGS_SETACCESSES, TAXIIAPI_SETCSVMAPPERS]}
```

Comment:
inside /dashboard/data/processing some part are allowed to KNOWLEDGE_KNUPDATE and it was missing, and some part are allowed to TAXIIAPI_SETCSVMAPPERS too.